content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/using.md

```diff
@@ -0,0 +1,98 @@
+---
+date: '2023-06-20'
+title: "Using the UTXO Ledger Token Selection API"
+project: corda
+version: 'Corda 5.0 Beta 4'
+menu:
+  corda5:
+    identifier: corda5-develop-utxo
+    parent: corda5-api-ledger-utxo-token-selection
+    weight: 1000
+section_menu: corda5
+---
```

Comment:
@blsemo Can you confirm this is still accurate for the latest version?

Reply:
Hi, we need to add a section for the token selection balance query. new content for this will be added to the original wiki here: https://r3-cev.atlassian.net/wiki/spaces/CB/pages/4302798863/Token+Selection+Overview

Reply:
Hi @owenstanford, I've added this section if you could review please: https://corda5.preview.docs.r3.com/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/token-selection/using.html